src/db/setups/expenses.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const Expenses = require('../models/expenses');
+
+Expenses.sync({ alter: true });
```

Comment:
Consider wrapping Expenses.sync() in an async function to handle errors and ensure successful execution 🤖